modules/data-entry/src/main/frontend/src/questionnaire/QuestionnaireStyle.jsx

```diff
@@ -627,6 +627,10 @@ const questionnaireStyle = theme => ({
       flexBasis: "100%",
       paddingLeft: theme.spacing(4)
     },
+    tableCell: {
+      textAlign: "middle",
```

Comment:
`middle` is not a supported value for `test-align`.
```suggestion
      textAlign: "center",
```